docs/chapter4-payload-handoff-format.rst

```diff
@@ -35,3 +35,139 @@ architecture, either 32 or 64 bits.
+
+.. tabularcolumns:: | p{3cm} p{0.75cm} p{2cm} p{9.5cm} |
+.. table:: Node: reserved-memory
+
+  ================ =========== ============ ======================================================
+   Property Name       Usage     Value Type      Definition
+  ================ =========== ============ ======================================================
+   #address-cells       R
+   #size-cells          R
+   ranges
+  ================ =========== ============ ======================================================
+
+Child Node:
+
+.. tabularcolumns:: | p{3cm} p{0.75cm} p{2cm} p{9.5cm} |
+.. table:: Node: name@xxx
+
+  ================ =========== ================ ======================================================
+   Property Name      Usage     Value Type       Definition
+  ================ =========== ================ ======================================================
+    reg                R        u32 / 64 array   Specify memory region of reserved memory
+    no-map             O        boolean          If present, indicates the operating system must not 
+                                                 create a virtual mapping of the region
+    compatible         O        string list
+  ================ =========== ================ ======================================================
+
+Child Nodes for Payload Memory Types:
+One of the example usage is that UEFI Payload will be aware of where pre-installed acpi tables or NVS regions are and will adopt them for supporting additional ACPI table installation from payload phase. Those boot-code/boot-data regions from PlatformInit will be reported to UEFI OS as usable memory without waste. Similarly runtime-code/runtime-data may be provided by PlatformInit for supporting UEFI runtime services that will be utilized by UEFI OS. All of the below are optional and can be skipped if unsupported by the platform. 
+
+
+  ================== ======================================================
+  Compatible string       Description
+  ================== ======================================================
+    acpi              Memory that holds ACPI tables
+    acpi-nvs          ACPI NVS buffer information
+    boot-code         Memory that holds firmware phase drivers and will be released to the OS when firmware boot phase finishes (for example with a UEFI payload when ExitBootService signaled).
+    boot-data         Memory that holds firmware phase data consumed by drivers and will be released to the OS when firmware boot phase finishes (for example with a UEFI payload when ExitBootService signaled).
+    runtime-code      Runtime service code memory region which will be utilized by OS runtime service
```

Comment:
How about 'used' instead of 'utilized' ?

Reply:
sure. it was fixed.